src/runtime/opencl/opencl_common.h

```diff
@@ -315,6 +315,13 @@ class OpenCLModuleNode : public ModuleNode {
   cl_kernel InstallKernel(cl::OpenCLWorkspace* w, cl::OpenCLThreadEntry* t,
                           const std::string& func_name, const KTRefEntry& e);
 
+  /*
+   * \brief Splits the provided serialized source file into separate
+   * source for each kernel primitive.
+   * \param source The serialized program source file (fmt: cl)
```

Comment:
document return

Reply:
gentle ping @csullivan

Reply:
Thanks for the ping @tqchen, should be good to go.